<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Corral√≥n Armitano</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
            QFrame{
            background-color:rgb(0,0,0);
            }

            QPushButton{
            background-color:rgb(61,61,61);
            border-radius:20px;
            color:rgb(255,255,255);
            font: 77 10pt &quot;Arial Black&quot;
            }

            QPushButton:hover{
            background-color:rgb(0,206,151);
            border-radius:20px;
            }

            QLabel{
            background-color:rgb(0,206,151);
            border-radius:5px;
            font: 77 10pt &quot;Arial Black&quot;
            }
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="lbl_cartel">
         <property name="minimumSize">
          <size>
           <width>340</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>340</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Procesador de listas de precios</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="btn_buscar">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Examinar...</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="btn_salir">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Salir</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
